classes/system/stacklight/server/cluster.yml

```diff
@@ -1,9 +1,45 @@
+applications:
+- keepalived
 classes:
 - system.elasticsearch.server.cluster
 - system.kibana.server.single
 - system.grafana.server.single
 - system.openstack.common.mk20_stacklight
 - service.grafana.collector
+- service.haproxy.proxy.single
+- service.keepalived.support
 parameters:
   _param:
     kibana_elasticsearch_host: ${_param:monitor_vip_address}
+  haproxy:
+    proxy:
+      listen:
+        elasticsearch_server:
+          check: true
```

Comment:
Please remove "check" to rely on the default value. This is to accommodate both StackLight and the legacy monitoring impl.